lean4-settings.el

```diff
@@ -18,16 +18,16 @@
   :prefix "lean4-"
   :group 'lean)
 
-(defvar-local lean4-default-executable-name
+(defconst lean4-default-executable-name
   (cl-case system-type
-    ('windows-nt   "lean.exe")
-    (t             "lean"))
+    (windows-nt "lean.exe")
```

Comment:
This one I found surprising, is this really correct?

Reply:
It is correct. You can try `eval-expression` with the following input to check it:

```emacs-lisp
(cl-case 'windows-nt
  (windows-nt "lean.exe")
  (otherwise "lean"))
```

It should print "lean.exe".

`cl-case` even accepts multiple values, so the following works too:

```emacs-lisp
(cl-case 'windows-nt
  ((windows-nt ms-dos) "lean.exe")
  (otherwise "lean"))
```

Reply:
I guess I'll never learn Elisp!